dashboard/config/locales/dsls.pt-BR.yml

```diff
@@ -1033,7 +1033,7 @@ pt-BR:
 
           ## Código Introduzido
 
-          *  **[Comment](//curriculum.code.org/documentation/weblab/comment/)** - `<!-- -->`
+          *  **[Comment](//studio.code.org/docs/weblab/comment/)** - `<!-- -->`
```

Comment:
note that these files are generated from crowdin content; the manual updates here and in `dashboard/config/locales/dsls.es-MX.yml` will be overridden the next time we sync, unless the appropriate translation is also updated